pages/common/wormhole.md

```diff
@@ -0,0 +1,23 @@
+# wormhole
+
+> Share files from one computer to another, safely.
+
+- Send a text message:
+
+`wormhole send`
+
+- Send a file/directory:
+
+`wormhole send path/to/file`
+
+- Receive a file/directory/text message:
```

Comment:
I don't understand this. If we can send only a file or directory, how can we get a text message also ?